packages/eas-cli/src/utils/queries.ts

```diff
@@ -26,10 +27,9 @@ type PaginatedQueryWithSelectPromptArgs<QueryReturnType extends Record<string, a
   BasePaginatedQueryArgs<QueryReturnType> & {
     promptOptions: {
       readonly title: string;
-      createDisplayTextForSelectionPromptListItem: (
-        queryItem: QueryReturnType
-      ) => SelectPromptEntry;
+      convertQueryItemToChoice: (queryItem: QueryReturnType) => SelectPromptEntry;
```

Comment:
```suggestion
      makeChoiceObject: (queryItem: QueryReturnType) => SelectPromptEntry;
```
```suggestion
      formatChoice: (queryItem: QueryReturnType) => SelectPromptEntry;
```

Reply:
I don't object to the change but wanted to point out that whatever is returned is not a real choice object, because getIdentifierForQueryItem returns a value, that is a part of that object., This is mostly why I decided to keep createDisplayTextForSelectionPromptListItem option.

Reply:
@wkozyra95 @dsokal What do you think about `makePartialChoiceObject`?

Reply:
Sounds good